### PR TITLE
Index routes

### PR DESCRIPTION
**Pull request by stamaleiferis**
Index routes: verficiation check , change password, forgot password
Staff routes: approve student, approve staff

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -37,4 +43,46 @@ router.post('/login', (req, res, next) => {
...
 
-module.exports = router;
\ No newline at end of file
+router.get('/verification/:email/:verificationToken',async (req, res, next) => {
+  const email = req.params.email
+  const verificationToken = req.params.verificationToken
+  try{
+    const dbData = await req.db.collection("User").find({email: email}).project({verificationToken:1,_id:0}).toArray();
+    if (verificationToken == dbData[0]['verificationToken']){
+      res.send({Success:true})
```
[the remark this answers is not in the document]
Comment (stamaleiferis) [reply routes/index.js:52]: User collection doesn't have emailVerified field. 
Would need to get role form User collection, then update Student/Staff collection.

OR put emailVerified field in User collectioj

OR verification happens on student and staff routes separately (already implemented)

File: routes/staff.js [hunk 2]
```diff
@@ -208,4 +208,31 @@ router.get('/status/:email', async (req,res)=>{
   }
 });
 
+router.post('/approve/student', async (req,res)=>{
+  const approve = req.body.approve
```
[the remark this answers is not in the document]
Comment (stamaleiferis) [reply routes/staff.js:212]: Done

File: routes/staff.js [hunk 3]
```diff
@@ -208,4 +208,31 @@ router.get('/status/:email', async (req,res)=>{
...
+  const email = req.body.email
+
+  try{
+    const dbData = await req.db.collection("Student").updateOne({email: email},{$set:{approved:approve}});
```
[the remark this answers is not in the document]
Comment (stamaleiferis) [reply routes/staff.js:216]: Yes let's send email, something like a welcome email maybe?

File: routes/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (reshadbinharun) [reply routes/index.js:52]: We could just use the `email` from the User record to find the Student/Staff  record, then update it. So the first option!

File: routes/staff.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (reshadbinharun) [reply routes/staff.js:216]: Yep! A welcome email.

**Pull request merged**